.golangci.yml

```diff
@@ -1,43 +1,21 @@
 linters:
   enable:
     - bodyclose
-    - deadcode
```

Comment:
why are those "unnecessary"? they seem like good defaults from Cosmos SDK? or is it they are covered by a different linter now

Reply:
These are enabled by default. No need to specify them. https://golangci-lint.run/usage/linters/